Bottom: eb7a9ad0d044f0452e9fd12d745f5dc7fc07dcb1
Top:    46443fdac177f3a6d9a4cd66b4e645ecf3cd501c
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-11 10:02:58 +0800

Refresh of autofs-5.0.7-make-dump-maps-check-for-duplicate-indirect-mounts.patch

---

diff --git a/lib/master.c b/lib/master.c
index 6143a9c..9e2ecfb 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1350,11 +1350,6 @@ int master_show_mounts(struct master *master)
 		while (source) {
 			struct mapent *me;
 
-			if (count && ap->type == LKP_INDIRECT)
-				printf("\n  duplicate indirect map source for"
-					" %s, will be ignored at run time\n",
-					ap->path);
-
 			if (source->type)
 				printf("\n  type: %s\n", source->type);
 			else {
@@ -1369,6 +1364,9 @@ int master_show_mounts(struct master *master)
 					printf("  map: %s\n", source->argv[0]);
 					i = 1;
 				}
+				if (count && ap->type == LKP_INDIRECT)
+					printf("  duplicate map source will be"
+						" ignored at run time\n");
 				if (source->argc > 1) {
 					printf("  arguments: ");
 					for (; i < source->argc; i++)
